python/res/enkf/ensemble_config.py

```diff
@@ -64,104 +64,104 @@ def __init__(self, config_content=None, grid=None, refcase=None, config_dict=Non
         
         c_ptr = None
         if config_dict is not None:
-            c_ptr = self._alloc_full(config_dict[ConfigKeys.GEN_KW_TAG_FORMAT])
+            c_ptr = self._alloc_full(config_dict.get(ConfigKeys.GEN_KW_TAG_FORMAT))
             if c_ptr is None:
                 raise ValueError("Failed to construct EnsembleConfig instance from dict")
 
             super(EnsembleConfig, self).__init__(c_ptr)
 
             gen_param_list = config_dict.get(ConfigKeys.GEN_PARAM, [])
             for a in gen_param_list:
```

Comment:
single letter variable names should be avoided. Maybe can we change this when we already is changing the access method?

Reply:
Which single letter variable you mean actually - c_ptr?